src/Cooler.sol

```diff
@@ -285,7 +285,7 @@ contract Cooler is Clone {
         collateral().safeTransfer(loan.lender, loan.collateral);
 
         // Log the event.
-        factory().newEvent(loanID_, CoolerFactory.Events.DefaultLoan, 0);
+        factory().logDefaultLoan(loanID_);
```

Comment:
The entry in the `loans` mapping is still being deleted, so the `loanID_` is useless.

I would add the amount of collateral transferred as a parameter to the event.

If you have that parameter, then it's possible to cache contract state earlier in the subgraph and access the loan data. But I think it's easier if the loan data is retained.